first_problem.rb

```diff
@@ -0,0 +1,17 @@
+class FirstProblem
+
+
+  def initialize(input_array)
+    @numbers_array = input_array
+
+  end
+
+  def compute()
+    sum = 0
+    @numbers_array.each do |number|
+      sum+= number if number.even?
```

Comment:
Ok this one option on how to do it, but ruby is really more expressive (as you've noticed with `even?`). Can you figure out how to do this in 1 line?

Reply:
i guess i might have done it !